compiler/semstmts.nim

```diff
@@ -241,6 +246,10 @@ proc semTry(c: PContext, n: PNode; flags: TExprFlags): PNode =
     elif a.kind != nkFinally:
       illFormedAst(n, c.config)
```

Comment:
This checks that the last node is a `finally`. Do you mean that only the last node can be a finally?